monai/handlers/roc_auc.py

```diff
@@ -33,10 +33,8 @@ class ROCAUC(IgniteMetric):  # type: ignore[valid-type, misc]  # due to optional
                 indicator matrix as a label.
             - ``"none"``: the scores for each class are returned.
 
-        output_transform: a callable that is used to transform the
-            :class:`~ignite.engine.Engine` `process_function` output into the
-            form expected by the metric. This can be useful if, for example, you have a multi-output model and
-            you want to compute the metric with respect to one of the outputs.
+        output_transform: callable to construct the `[y_pred, y]` pair from `ignite.engine.state.output`,
```

Comment:
Same suggestion as written in `ignite_metric.py`.

Reply:
updated in the latest commit.
Thanks.